hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/5_5_0/2845-reindex-operation.yaml

```
@@ -0,0 +1,4 @@
+---
+type: add
+issue: 2845
+title: "Added new $reindex operation with similar syntax to $delete-expunge that creates a spring-batch job to reindex selected resources."
```

This is a bit light for the size of the change.  
```suggestion
title: "Added new `$reindex` operation with similar syntax to `$delete-expunge` that creates a spring-batch job to reindex selected resources.
`$mark-all-resources-for-reindexing` and `$perform-reindexing-pass` are now deprecated, and will likely be removed in a future release."
```